### ISSUE TITLE
Attempt to invoke virtual method 'an droid.content.res.Resources android.view.View.getResources()' on a null object reference

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#2045]
**Summary:** 

Got a crash when starting the app.

**Steps to reproduce:** 

Not easy to reproduce, but second time today.

**System logs:**

```
11-29 22:39:35.025 23369 23517 D NoSync  : Called fsync                                                                                
11-29 22:39:35.029 23592 23809 I native  : flash-index.cc:1605 Flash index sort hb 11198 prefix-hits 49416 -> 33239 at 539.188ms
11-29 22:39:35.101 23592 11637 I iu.FingerprintManager: Start processing media store URI: content://media/external/video/media            
11-29 22:39:35.169 12009 12009 D NearbyFragment: On nearby tab selected                                        
11-29 22:39:35.207   794   456 I GnssLocationProvider: WakeLock acquired by sendMessage(3, 0, com.android.server.location.GnssLocationProvider$GpsRequest@1a6fa98)
11-29 22:39:35.210 12009 12009 D NearbyFragment: checking GPS                                                 
11-29 22:39:35.210 12009 12009 D NearbyFragment: GPS is enabled                                                                                                                                                                    
11-29 22:39:35.211 12009 12009 D NearbyFragment: Checking location permission                                                                                                                                                                           
11-29 22:39:35.212 12009 12009 D NearbyFragment: Refreshing nearby places                                                                                           
11-29 22:39:35.214 12009 12009 D NearbyFragment: Skipping update of nearby places as location is unavailable 
11-29 22:39:35.217 12009 12009 D AndroidRuntime: Shutting down VM                                             
11-29 22:39:35.218 12009 12009 E AndroidRuntime: FATAL EXCEPTION: main                      
11-29 22:39:35.218 12009 12009 E AndroidRuntime: Process: fr.free.nrw.commons, PID: 12009   
11-29 22:39:35.218 12009 12009 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{fr.free.nrw.commons/fr.free.nrw.commons.contributions.MainActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'an
droid.content.res.Resources android.view.View.getResources()' on a null object reference                                                                                                                          
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2684)
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2751)                        
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at android.app.ActivityThread.-wrap12(ActivityThread.java)
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1496)                                                     
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:102)
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:154)
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:6186)                             
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
11-29 22:39:35.218 12009 12009 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.res.Resources android.view.View.getResources()' on a null object reference
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at android.support.design.widget.Snackbar.make(Snackbar.java:170)                                                                           
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at fr.free.nrw.commons.nearby.NearbyFragment.addNetworkBroadcastReceiver(NearbyFragment.java:596)                                                                                                    
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at fr.free.nrw.commons.nearby.NearbyFragment.performNearbyOperations(NearbyFragment.java:639)                                               
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at fr.free.nrw.commons.nearby.NearbyFragment.onTabSelected(NearbyFragment.java:627)                
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at fr.free.nrw.commons.contributions.MainActivity.onCreate(MainActivity.java:87)
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at android.app.Activity.performCreate(Activity.java:6684)                                                                                 
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1119)                                             
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2637)
11-29 22:39:35.218 12009 12009 E AndroidRuntime:        ... 9 more                                                                                                                                                           
11-29 22:39:35.238   794  3630 W ActivityManager:   Force finishing activity fr.free.nrw.commons/.contributions.MainActivity
```

**Device and Android version:** 

LineageOS Android 7.1.2
 
**Commons app version:** 

2.9.0 from Play Store

**Comment by misaochan** [comment]
It seems the PR was closed prematurely?

**Comment by maskaravivek** [comment]
@misaochan Have just reopened the PR.

**Comment by albendz** [comment]
Fix has been merged with #2046
